### PR TITLE
CPU and Memory Quota should not include series from workloads that have released resources

### PR DESCRIPTION
**Pull request by PhilipGough**
The bug was raised downstream in https://bugzilla.redhat.com/show_bug.cgi?id=1913618

Dashboards were displaying cpu and memory requests and limits for pods and jobs that had completed or failed and hence skewed the quotas on the dashboards

cc @paulfantom

### CODE REVIEW

File: rules/apps.libsonnet [hunk 1]
```diff
@@ -100,6 +127,15 @@
               )
             ||| % $._config,
           },
+          {
+            record: 'container_cpu:kube_pod_container_resource_limits',
```
Comment (paulfantom) [inline rules/apps.libsonnet:131]: nit: I am not exactly sure about the prefix. @metalmatze @povilasv any takes on this?

Comment (brancz) [reply rules/apps.libsonnet:131]: Agreed, I think is should be something like `cluster:namespace:pod:active:kube_pod_container_resource_limits`

Comment (povilasv) [reply rules/apps.libsonnet:131]: :+1: I like @brancz suggestion, let's rename it.

Comment (paulfantom) [reply rules/apps.libsonnet:131]: I don't agree, with name `cluster:namespace:pod:active:kube_pod_container_resource_limits` we are loosing information that this is about CPU and we have immediate conflict with similar rule for memory - https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/639/files#diff-bc8c24b130076f1ca75adec5baee841ea4bed3a51a5d72b4437e31dd1d188a5cR108-R115

Comment (PhilipGough) [reply rules/apps.libsonnet:131]: How about `cluster:namespace:pod_cpu:active:kube_pod_container_resource_limits `, `cluster:namespace:pod_memory:active:kube_pod_container_resource_limits ` ?

Comment (brancz) [reply rules/apps.libsonnet:131]: Whops, yes my bad, cpu should definitely be in there. I'd be happy with your latest suggestion @PhilipGough!

**Comment by PhilipGough** [comment]
@brancz @paulfantom amended as discussed

**Review by brancz — APPROVED** [review]
lgtm but giving @paulfantom another chance to review

**Pull request merged**